### PR TITLE
Rename ResNet50 to latest Metalhead

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**
Updates Metalhead's new name for ResNet. This is only needed in the tests.

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -38,7 +38,7 @@ end
 
 @testset "ResNet test" begin
   # addprocs(2, exeflags = "--project=.")
-  resnet = ResNet50()
+  resnet = ResNet()
```
Comment (CarloLucibello) [inline test/runtests.jl:41]:
```suggestion
  resnet = ResNet(50)
```

**Comment by DhairyaLGandhi** [comment]
Failure here looks real, at least for Julia 1.6, which we should bump anyway

**Comment by DhairyaLGandhi** [comment]
@jpsamaroo mind if i merge this in your branch?

**Comment by jpsamaroo** [comment]
Yeah feel free to!

**Pull request merged**